lib/core/cfg.js

```diff
@@ -1,19 +1,29 @@
 const fs = require('fs-extra');
-
 const { Files } = require('./files');
-
 const files = new Files();
+const process = require('process');
 
 module.exports = {
-  read: (cfgPath) => {
-    // check permissions of the config file
-    const stat = fs.statSync(cfgPath);
+  read: (path) => {
+    module.exports.checkPermissions(path)
```

Comment:
I really don't like this syntax, but "this" is not working; do you know a better way to reference a function inside the same module.exports?

Reply:
Nope, the same thing is used all over the place in the project... At some point it might be worth the effort to spend some time migrating this to typescript, all would be much better. We could start by using the existing config, log, helm, etc. ts packages, and just leave here the specific logic. We can talk about this later.